package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/publitas/react-to-svg-component#readme",
   "dependencies": {
-    "babel-core": "^6.26.3",
+    "babel-core": "^4.7.16",
```

Comment:
just double checking: this looks like it was downgraded instead of upgraded

Reply:
Oh, good catch, @felixhageloh! I hadn't noticed that.
I wonder why npm is doing that. It is not the first time. I will fix that.